docs/section4/4-3.md

```diff
@@ -0,0 +1,46 @@
+# ASHVAC – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. Not applicable for for heating or cooling system is being simulated only to meet the requirements described in Table G3.1.
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each HVAC system in P_RMR: `for hvac_p in P_RMR...heating_ventilation_air_conditioning_systems:`
+    - Get HVAC system status: `status_type_p = hvac_p.status_type:`
+        - Check if HVAC system status is new or existing: `if status_type_p in [NEW,EXISTING]:`
+            - Set applicability flag: `rule_applicability_check = TRUE`
+				- For each fan system in each HVAC system that meets the applicabliity check: `fan_systems_p in heating_ventilation_air_conditioning_systems_p.fan_systems:`
+					- Get minimum OA: `minimum_outdoor_airflow_p = fan_systems_p. minimum_outdoor_airflow`
+					- Get operation during occupied periods: `operation_during_occupied_p = fans_systems_p.operation_during_occupied`
+
+**Rule Assertion:**
+- Case 1: if minimum OA is greater than 0 and fan system operation during occupied periods is continuous in the P_RMR: `if minimum_outdoor_airflow_p >0 and operation_during_occupied_p == “CONTINUOUS”: PASS`
+
+- Case 2: Else if minimum OA is greater than 0 and fan system operation during occupied periods is not set to CONTINUOUS P_RMR: `else if minimum_outdoor_airflow_p > 0 and operation_during_occupied_p == “KEEP_OFF”: FAIL and raise_warning “FAN SYSTEM IS OFF DURING OCCUPIED PERIODS WHEN THE UNIT IS MODELED AS PROVIDING MINUMUM OA CFM IN THE P_RMR.”`
+
```

Comment:
I think if the unit does not provide minimum OA then it can cycle with heating and cooling loads even during occupied periods. Right? That is how I interpret the language. This would apply to a unit heater serving stairs for example.